src/content/docs/guides/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -1156,6 +1156,16 @@ On the JavaScript side, the `WebviewWindow` class is now exported in the `@tauri
 
 The `onMenuClicked` function was removed, you can intercept menu events when creating a menu in JavaScript instead.
 
+### Migrate Embedded External Binaries (Sidecar)
+
+In Tauri v1, the external binaries and their arguments were defined in the allowlist in the Tauri config. In v2, you use the new permissions system to do so instead. Read [Migrate Permissions](#migrate-permissions) for more information. Alternatively, if you are upgrading from v1, the `migrate` command in Tauri v2 CLI should handle this for you. Read [Automated Migration](#automated-migration) for more information.
+
+On the JavaScript side, in v1, the sidecar was called using `import { Command } from '@tauri-apps/api/shell'`. In v2, you use the shell plugin instead. Read [Migrate to Shell Plugin](#migrate-to-shell-plugin) to see how.
```

Comment:
```suggestion
On the JavaScript side, make sure you [Migrate to Shell Plugin](#migrate-to-shell-plugin).
```
I don't think that much details is needed as it already covered in the migrate to shell plugin.